br/pkg/lightning/backend/external/util.go

```diff
@@ -211,7 +211,7 @@ const (
 type Endpoint struct {
 	Key    []byte
 	Tp     EndpointTp
-	Weight uint64 // all EndpointTp use positive weight
+	Weight int64 // all EndpointTp use positive weight
```

Comment:
The comment metioned "all EndpointTp use positive weight", why here change to int64?